javalib/src/main/scala/java/net/URI.scala

```diff
@@ -54,7 +54,7 @@ final class URI(origStr: String) extends Serializable with Comparable[URI] {
   private val _authority = fld(AbsAuthority, RelAuthority).filter(_ != "")
   private val _userInfo = fld(AbsUserInfo, RelUserInfo)
   private val _host = fld(AbsHost, RelHost)
-  private val _port = fld(AbsPort, RelPort).fold(-1)(_.toInt)
+  private val _port = fld(AbsPort, RelPort)
```

Comment:
`_port` should still be an `Int` field (add `.map(_.toInt)`)

Reply:
Addressed in https://github.com/scala-js/scala-js/pull/3690/commits/c0eaf768aa58dde27f84927961440e898479bf33